proxies/live/apiproxy/targets/sync-wrap.xml

```diff
@@ -40,6 +40,13 @@
         <Condition>(response.status.code Equals "503")</Condition>
          <Name>AssignMessage.Errors.CatchAllMessage</Name>
       </Step>
+      <!-- APMSPII-938 explicitly convert 422 error from spine to 500 unkown error as
+        as 422 errors will have no meaning for the end user as the implementation details
+        of sync-wrap are hidden. -->
+      <Step>
+        <Condition>(response.status.code Equals "422")</Condition>
```

Comment:
Is this essential, I can vaguely remember we wanted to avoid to raising faults here since you'd have to keep the error message sync'd with spine error messages?